tests/test_note.py

```diff
@@ -556,3 +556,53 @@ def test_note_to_from_nonimpl(meth):
 def test_abc_doctave_calc(note, oct, expected):
     fn = partial(_octave_from_abc_parts, base=0)
     assert fn(note, oct) == expected
+
+
+@pytest.mark.parametrize(
+    "scientific,expected_helmholtz",
+    [
+        ("C1", "C,"),
+        ("C2", "C"),
+        ("C3", "c"),
+        ("C4", "c'"),
+        ("C5", "c''"),
+        ("B3", "b"),
+        ("B#3", "b#"),
+        ("D3", "d"),
+    ],
+)
+def test_helmholtz_from_pitch(scientific, expected_helmholtz):
+    pitch = Pitch.from_name(scientific)
+    assert pitch.helmholtz() == expected_helmholtz
```

Comment:
This is probably sufficient, but could also check that they have the same `.name`.

Reply:
I don't think I understand the exact request here. `expected_helmholtz` is just a string, so it wouldn't have a `.name` property. I could check the pitch class? I'll also add a c-flat example, since it's another good edge case...